src/main/java/uk/gov/hmcts/reform/lrdapi/util/ValidationUtils.java

```diff
@@ -0,0 +1,140 @@
+package uk.gov.hmcts.reform.lrdapi.util;
+
+import org.apache.commons.lang.StringUtils;
+import org.slf4j.Logger;
+import uk.gov.hmcts.reform.lrdapi.controllers.advice.InvalidRequestException;
+import uk.gov.hmcts.reform.lrdapi.controllers.constants.LocationRefConstants;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class ValidationUtils {
+
+    private ValidationUtils() {
+
+    }
+
+    private static final String REG_EXP_SPCL_CHAR = "^[^<>{}\"/|;:.~!?@#$%^=&*\\]\\\\()\\[¿§«»ω⊙¤°℃℉€¥£¢¡®©09+]*$";
+    private static final String REG_EXP_WHITE_SPACE = "\\s";
+    private static final String EXCEPTION_MSG_SPCL_CHAR = "Param contains special characters. "
+        + "',' comma and '_' underscore allowed only";
+    private static final String EXCEPTION_MSG_ONLY_ONE_OF_THREE_PARAM = "Please provide only 1 of 3 params: "
+        + "serviceCode, ccdCaseType, ccdServiceNames.";
+    private static final String REG_EXP_COMMA_DILIMETER = ",(?!\\\\s)";
+
+    public static boolean validateInputParameters(String serviceCode, String ccdCaseType, String ccdServiceNames) {
```

Comment:
all constants can be moved to one constant file

Reply:
This has been done